src/components/LabelMenus/styles.module.css

```diff
@@ -16,12 +16,40 @@
   position: relative;
 }
 
-.highlightLabel {
-  border-bottom: 0.25rem solid black;
+.highlightLabelUseCases {
+  background-color: black;
+  display: inline-block;
+  text-align: center !important;
+  font-weight: bold;
+  color: white;
+  /* border-bottom: 0.25rem solid black; */
   cursor: pointer;
-  padding-bottom: 0.3125rem;
+  padding: 0.3125rem 0rem 0.3125rem 0rem;
+  width: 123%;
 }
 
+.highlightLabelDataset {
+  background-color: black;
+  display: inline-block;
+  text-align: center !important;
+  font-weight: bold;
+  color: white;
+  /* border-bottom: 0.25rem solid black; */
+  cursor: pointer;
+  padding: 0.3125rem 0rem 0.3125rem 0rem;
+  width: 136%;
+}
+.highlightLabelOverview {
+  background-color: black;
+  display: inline-block;
+  text-align: center !important;
+  font-weight: bold;
+  color: white;
+  /* border-bottom: 0.25rem solid black; */
+  cursor: pointer;
+  padding: 0.3125rem 0rem 0.3125rem 0rem;
+  width: 147%;
+}
```

Comment:
All 3 of these are pretty much the same it seems, expect for the width, right? I wonder if we can find a way to not need the width specified, then these can collapse into one class.

If they end up needing the width specified, then we can still make this a bit simpler by having everything except `width` be in one class called `.highlightLabel`, and then three separate other classes that specify width only (so that we don't need to change the other stylings in three places, should they need changes in the future).

Reply:
done